src/main/java/seedu/agendum/logic/parser/Parser.java

```diff
@@ -32,6 +33,8 @@
 
     private static final Pattern RENAME_ARGS_FORMAT = Pattern.compile("(?<targetIndex>\\d+)\\s+(?<name>[^/]+)");
 
+    private static final Pattern ADD_ARGS_FORMAT = Pattern.compile("(?<name>[^/]+)(\\/from\\s)(?<fromArg>\\b.+?(?=\\/|$)){0,1}(\\/to\\s)(?<toArg>\\b.+?(?=\\/|$)){0,1}");
```

Comment:
Problem with the regex is that it's not taking the 2 regex's as optionals, even thought I ended it with a `{0,1}`. So if  it only finds a `/from` and not a `/to`, matcher.matches() fails.

That should not be the case. We need them to be optional. i.e. you could enter `/from`, `/to` or nothing at all.